src/styles/blocks/main.scss

```diff
@@ -0,0 +1,21 @@
+.main {
+  margin: 0;
+  padding: 50px 40px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it is recommended to use logical properties for margin and padding to support different writing modes. Consider using `margin-inline` and `padding-block` instead of `margin` and `padding`.